src/comments/handlers/MemberAccess.js

```diff
@@ -0,0 +1,19 @@
+const {
+  util: { addDanglingComment }
+} = require('prettier');
+
+function handleMemberAccessComments(
+  text,
+  precedingNode,
+  enclosingNode,
+  followingNode,
+  comment
```

Comment:
not all these params have been used; maybe pass an object into the function instead of all these params?